controls/V-72273.rb

```diff
@@ -58,20 +65,11 @@
 # systemctl start firewalld"
   tag "fix_id": "F-78627r1_fix"
 
-  describe.one do
-    describe package('firewalld') do
-      it { should be_installed }
-    end
-    describe package('iptables') do
-      it { should be_installed }
-    end
+  describe package(firewall_application) do
+    it { should be_installed }
   end
-  describe.one do
```

Comment:
IPTables is still valid per the general prose.

Reply:
Correct, this will pass if either firewalld or iptables is installed and running. I am not sure what change needs to be made here.

Reply:
@djhaynes I think that the previous version was correct. The new version will fail if `firewalld` is removed and/or disabled but, per the prose, that is valid as long as `iptables` is configured correctly.

Reply:
Reverted back to the previous version.